swift/read_json_file.swift

```diff
@@ -0,0 +1,18 @@
+//Read JSON file, transform and print to stdout
+import Foundation
+
+guard let jsonFile = CommandLine.arguments.dropFirst().first else {
+    print("Usage: swift script.swift <json_file>")
+    exit(1)
+}
+
+let fileURL = URL(fileURLWithPath: jsonFile)
+let jsonData = try Data(contentsOf: fileURL)
+let people = try JSONSerialization.jsonObject(with: jsonData) as? [[String: Any]] ?? []
+
+for (index, person) in people.enumerated() {
+    if let age = person["age"] as? Int, let firstName = person["first_name"] as? String {
+        print("Hello, \(age) year old \(firstName)")
+    }
+}
```

Comment:
```suggestion
    let age = person["age"] as! Int
    let firstName = person["first_name"] as! String

    print("Hello, \(age) year old \(firstName)")
```

Failing immediately instead of handling the error with an if statement. 

Let's apply this strategy to the rest of the swift scripts, too.